clusterloader2/testing/density/config.yaml

```diff
@@ -6,6 +6,14 @@
 {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
 {{$TEST_THROUGHPUT := DefaultParam .TEST_THROUGHPUT 20}}
+# LATENCY_POD_MEMORY is calculated for 1cpu node. The scheduler assigns 1 point
+# per 10% unallocated cpu quata. 0.1 cpu per latency pod should change
+# scheduling priority with every latency pods assignment.
+{{$LATENCY_POD_CPU := DefaultParam .LATENCY_POD_CPU 100}}
+# LATENCY_POD_MEMORY is calculated for 4GB memeory node. In this scenario, memory is not a issue,
```

Comment:
nit - can you please merge those two comments? Sth like:

Latency pods cpu/memory is calculated for 1-core 4GB node. Decreasing allocation of both memory and cpu by 10% increases the value of priority function in scheduler by one point, resulting in decreasing priority of choosing the same node again.

[or sth like that]

Reply:
Done.